data/remnant/remnant 2 cognizance.txt

```diff
@@ -657,7 +657,7 @@ mission "Remnant: Cognizance 18"
 		event "remnant: pantica void sprites"
 		event "remnant: cognizance timer" 30
 		conversation
-			`Moments later a ping comes in on the commlink, and Prefect Chilia and Plume appear on screen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in," concludes Plume.`
+			`Moments later a ping comes in on the commlink, and Prefect Chilia and Plume appear on-screen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in," concludes Plume.`
```

Comment:
See https://github.com/endless-sky/endless-sky/pull/10590#discussion_r1804541079.
Right now, without an explanation, this is literally just churn.

Reply:
I made this change before you made that comment. 
```suggestion
			`Moments later a ping comes in on the commlink, and Prefect Chilia and Plume appear on screen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in," concludes Plume.`
```

Reply:
I personally would prefer that we remove the space here. Whether it goes to onscreen or on-screen I am on the fence about, but the space just doesn't look or flow good for me. Honestly, even "on the screen" would work better than just "on screen" (but worse than "onscreen" or "on-screen")